docs/spec/resource-scopes.md

```diff
@@ -144,4 +144,5 @@ module myMod './path/to/module.bicep' = {
 }
 ```
 
-[arm-scopes]: https://docs.microsoft.com/azure/azure-resource-manager/management/overview#understand-scope
\ No newline at end of file
+⮴📖 [resourceGroup properties] https://docs.microsoft.com/en-us/azure/templates/microsoft.resources/resourcegroups?tabs=bicep
+⮴📖 [arm-scopes]: https://docs.microsoft.com/azure/azure-resource-manager/management/overview#understand-scope
```

Comment:
I think these emoji are screwing up the link declaration. I had to remove them and add a newline between L147 and L148 to get this link to show up correctly.